### DISCUSSION TITLE
어질어질한 리드미 작성 팁

### DISCUSSION THREAD

**Discussion by saejinpark** [DeveloperAcademy-POSTECH/.github#18] [Idea]
# [GFM(GitHub Flavored Markdown)](https://github.github.com/gfm)

> CommonMark의 엄격한 상위 집합
> GitHub 사용자 콘텐츠에서 지원됨
> 원래 CommonMark 사양에 지정되지 않은 모든 기능은 **확장 기능**
> GFM은 광범위한 입력을 지원
> 보안과 일관성을 보장하기 위해 GFM이 HTML로 변환된 후 추가 사후 처리 및 정리 작업을 수행

## 예시
```
# Heading
<h1>Heading</h1>
```
### 결과
# Heading
<h1>Heading</h1>

```
...
흔한 내용들 생략
```
### [stackedit](https://stackedit.io/): 여기서 일반적인 것들은 알 수 있어요
---
# 작성 팁
> `width` 블록요소에 너비(높이 X)를 고정값으로 조정가능
<table>
    <tr>
        <td width="100">100</td>
    </tr>
</table>
<table>
    <tr>
        <td width="200">200</td>
    </tr>
</table>
<table>
    <tr>
        <td width="300">300</td>
    </tr>
</table><table>
    <tr>
        <td width="400">400</td>
    </tr>
</table>
<table>
    <tr>
        <td width="500">500</td>
    </tr>
</table>

<details>
    <summary>코드</summary>

```
<table>
    <tr>
        <td width="100">100</td>
    </tr>
</table>
<table>
    <tr>
        <td width="200">200</td>
    </tr>
</table>
<table>
    <tr>
        <td width="300">300</td>
    </tr>
</table><table>
    <tr>
        <td width="400">400</td>
    </tr>
</table>
<table>
    <tr>
        <td width="500">500</td>
    </tr>
</table>
```

</details>

> `align`: 텍스트 정렬 변환[`left(default)`, `center`, `right`]
<table>
    <tr>
        <td align="left" width="300">text</td>
    </tr>
</table>
<table>
    <tr>
        <td align="center" width="300">text</td>
    </tr>
</table>
<table>
    <tr>
        <td align="right" width="300">text</td>
    </tr>
</table>

<details>
    <summary>코드</summary>

```
<table>
    <tr>
        <td align="left" width="300">text</td>
    </tr>
</table>
<table>
    <tr>
        <td align="center" width="300">text</td>
    </tr>
</table>
<table>
    <tr>
        <td align="right" width="300">text</td>
    </tr>
</table>
```

</details>

> 들여쓰기가 하고싶을 때 `ul` 활용
<ul>

text

</ul>

<details>
    <summary>열기</summary>

```
<ul>

text

</ul>
```

</details>

> `rowspan/colspan`: 테이블 잘 만들고 싶을 때
<table>
    <caption>[테이블 설명]</caption>
    <colgroup>
        <col />
        <col />
        <col />
    <colgroup>
    <tr>
        <th colspan="2">colspan 2, rowspan 1</th>
        <th rowspan="2">colspan 1, rowspan 2</th>
    </tr>
    <tr>
        <td>colspan 1, rowspan 1</td>
        <td>colspan 1, rowspan 1</td>
    </tr>
</table>

<details>
   <summary>열기</summary>

```
<table>
    <caption>[테이블 설명]</caption>
    <colgroup>
        <col />
        <col />
        <col />
    <colgroup>
    <tr>
        <th colspan="2">colspan 2, rowspan 1</th>
        <th rowspan="2">colspan 1, rowspan 2</th>
    </tr>
    <tr>
        <td>colspan 1, rowspan 1</td>
        <td>colspan 1, rowspan 1</td>
    </tr>
</table>
```

</details>

> readme에서 테두리 있는 블럭을 사용하고 싶다: `table`을 활용
<table>
    <colgroup>
        <col />
        <col />
        <col />
    </colgroup>
    <thead>
        <tr>
            <th colspan="3">header</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <th colspan="3">visual</th>
        </tr>
        <tr>
            <th>aside</th>
            <th>main</th>
            <th>aside</th>
        </tr>
    </tbody>
    <tfoot>
        <tr>
            <th colspan="3">footer</th>
        </tr>
    </tfoot>
</table>

<table>
    <tr>
        <td>
            <p>A</p>
            <table>
                <tr>
                    <td>
                        <p>B</p>
                        <table>
                            <tr>
                                <td>
                                    <p>C</p>
                                </td>
                            </tr>
                        </table>
                    </td>
                </tr>
            </table>
        </td>
    </tr>
</table>

> ### 깨알 철학
> `3단논법`: B가 A에 속하고 C가 B에 속하면 C는 A에 속한다.

<details>
    <summary>코드</summary>

```
<table>
    <colgroup>
        <col />
        <col />
        <col />
    </colgroup>
    <thead>
        <tr>
            <th colspan="3">header</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <th colspan="3">visual</th>
        </tr>
        <tr>
            <th>aside</th>
            <th>main</th>
            <th>aside</th>
        </tr>
    </tbody>
    <tfoot>
        <tr>
            <th colspan="3">footer</th>
        </tr>
    </tfoot>
</table>
<table>
    <tr>
        <td>
            <p>A</p>
            <table>
                <tr>
                    <td>
                        <p>B</p>
                        <table>
                            <tr>
                                <td>
                                    <p>C</p>
                                </td>
                            </tr>
                        </table>
                    </td>
                </tr>
            </table>
        </td>
    </tr>
</table>
```

</details>

빌드업 끝

머메이드 스터디 만들고 싶은데 만들면 같이 해주실 분 있나요?

**Comment by jinccc97** [comment]
우와 감사합니다!
